config/merge/merge_test.go

```diff
@@ -329,8 +330,8 @@ func TestMerge(t *testing.T) {
 					Luks: []types.Luks{
 						// nested struct pointers, one override
 						{
-							Clevis: &types.Clevis{
```

Comment:
Let's drop this entire test from `TestMerge`.  It doesn't serve a purpose with the new spec.